modules/accounting/assets/src/admin/components/products/ImportModal.vue

```diff
@@ -0,0 +1,465 @@
+<template>
+    <div id="wperp-product-import-modal">
+        <div class="wperp-container">
+            <div id="wperp-import-customer-modal" class="wperp-modal has-form wperp-modal-open" role="dialog">
+                <div class="wperp-modal-dialog">
+                    <div class="wperp-modal-content">
+                        <div class="wperp-modal-header">
+                            <h3>{{ __('Import Products', 'erp') }}</h3>
```

Comment:
Will we use space in Component file translation or not ? Some parts uses space, some not. But it should same for all.
```js
{{ __('Import Products', 'erp') }}
```
or

```js
{{ __( 'Import Products', 'erp' ) }}
```

Reply:
`{{ __('Import Products', 'erp') }}`
This form is more convenient for javascript

Reply:
`{{ __('Import Products', 'erp') }}`
This form is more convenient for javascript